src/domains/ecommerce-app/api/ecommerce-checkout/v2/_openapi.json.tpl

```diff
@@ -28,6 +28,77 @@
     "description": "Design review"
   },
   "paths": {
+    "/payment-methods/{id}/sessions": {
+      "post": {
+        "tags": [
+          "ecommerce-methods"
+        ],
+        "operationId": "createSession",
+        "summary": "Create frontend field data paired with a payment gateway session",
+        "description": "This endpoint returns an object containing data on how a frontend can build a form\nto allow direct exchanging of payment information to the payment gateway without eCommerce\nhaving to store PCI data (or other sensitive data tied to the payment method).\nThe returned data is tied to a session on the payment gateway identified by the field `orderId`.",
+        "parameters": [
+          {
+            "name": "id",
+            "in": "path",
+            "description": "Payment Method id",
+            "required": true,
+            "schema": {
+              "type": "string"
+            }
+          },
+          {
+            "in": "query",
+            "name": "recaptchaResponse",
+            "description": "Recaptcha response",
+            "schema": {
+              "type": "string"
+            },
+            "required": true
+          },
+		  {
+            "in": "header",
+            "name": "lang",
+            "required": true,
```

Comment:
Shouldn't this be optional?

Reply:
Yes as you can see from previous commits it was initially optional,
but i had to set it as required due to an issue with "yarn generate" on the checkout repo wich caused the optional language header to be translated into code with a bug in it (it would not compile due to sintax).